roles/zabbix_proxy/tasks/Debian.yml

```diff
@@ -136,25 +126,56 @@
       Pin: origin repo.zabbix.com
       Pin-Priority: {{ zabbix_proxy_apt_priority }}
     owner: root
-    mode: "0644"
+    mode: '0644'
   when:
     - zabbix_proxy_apt_priority | int
   become: true
 
+- name: apt-get clean
+  shell: apt-get clean; apt-get update
+  args:
+    warn: false
+  changed_when: false
+  become: true
+  tags:
+    - skip_ansible_lint
```

Comment:
```suggestion
- name: apt-get clean
  apt:
    clean: yes
    update_cache: yes
    cache_valid_time: 0
  become: true
  tags:
    - skip_ansible_lint
```

is this better?

Reply:
Hello of course, just blindly copied server role here. Ansible bindings for apt do the trick in a cleaner way.